src/uct/tcp/tcp_md.c

```diff
@@ -16,6 +16,7 @@ static ucs_status_t uct_tcp_md_query(uct_md_h md, uct_md_attr_t *attr)
 {
     /* Dummy memory registration provided. No real memory handling exists */
     attr->cap.flags               = UCT_MD_FLAG_REG |
+                                    UCT_MD_FLAG_INVALIDATE |
```

Comment:
but TCP does not really support invalidate?

Reply:
TCP doesn't support rkey actually, it is just no-op.
so, we could do invalidate for TCP to support RNDV for TCP

Reply:
IMO, the opposite: what prevents from sending PUT request to a released buffer?

Reply:
yes, need to add support in TCP in the near future